{{cookiecutter.project_name}}/Makefile

```diff
@@ -239,20 +239,35 @@ clean-all: clean clean-env .clean-workspace
 
 # Release ######################################################################
 
-.PHONY: register
-register: doc
-	$(PYTHON) setup.py register
-
 .PHONY: dist
```

Comment:
I think this target can go away now in favor of running `upload-test`. Can you update the README as well?